rust_dev_preview/eks/src/bin/create-delete-cluster.rs

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use aws_config::meta::region::RegionProviderChain;
+use aws_sdk_eks::model::VpcConfigRequest;
+use aws_sdk_eks::{Client, Error, Region, PKG_VERSION};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// The AWS Region.
+    #[structopt(short, long)]
+    region: Option<String>,
+
+    /// The name of the cluster.
+    #[structopt(short, long)]
+    cluster_name: String,
+
+    /// ARN for the role for the cluster
+    /// To create an ARN:
+    ///
+    /// 1. Follow instructions to create an IAM role:
+    /// https://docs.aws.amazon.com/eks/latest/userguide/service_IAM_role.html
+    ///
+    /// 2. Copy role ARN
```

Comment:
Nit: This feels like an incomplete statement

Reply:
I pulled these comments as we don't want links that could change.